site/en/docs/extensions/mv3/architecture-overview/index.md

```diff
@@ -2,370 +2,120 @@
-
-### UI elements {: #pages }
-
-An extension's user interface should be purposeful and minimal. The UI should customize or enhance
-the browsing experience without distracting from it. 
-
-The following is a list of the most common UI examples:
-
-- An [action click][docs-click] event.
-- A [popup][docs-popup].
-- A [context menu][docs-context-menu].
-- An [omnibox][docs-omnibox].
-- A [keyboard shortcut][docs-commands].
-- Desktop [notifications][api-notif].
-- [Text-to-speech][api-tts].
-- A custom UI injected [into a page][docs-content-scripts].
-
-See [Design the UI of a Chrome extension][docs-ui], to learn more.
-
-### Options page {: #optionsPage }
-
-Just as extensions allow users to customize the Chrome browser, the options page enables
-customization of the extension. Options can be used to enable features and allow users to choose
-what functionality is relevant to their needs.
-
-Users can access the options page via [direct link][docs-link-options] or in the context menu of the
-extension toolbar. The following is an example of the Google Dictionary extension. 
-
-{% Columns %}
-
-{% Column %}
-
-<figure>
-{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/Mz7GV76tFkzxRlb7Pq6e.png", 
-alt="Options page link in the UI", width="800", height="299" %}
-  <figcaption>
-    Link to the Options page.
-  </figcaption>
-</figure>
-
-{% endColumn %}
-
-{% Column %}
-
-<figure>
-{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/BM11QeGCThsUNTlsZbAe.png", 
-alt="Context Menu Options page", width="357", height="222" %}
+The manifest is the configuration file of a Chrome extension. It is a required JSON file that must be located at the [root of the project][dev-basics-structure]. It provides the browser with a blueprint of the extension, with important information such as:
 
-  <figcaption>
-    Options page in the extension's context menu.
-  </figcaption>
-</figure>
+- The name of the extension, a description of what it does, the current version number, and what icons to use.
+- The [Chrome API][api-ref] keys and [permissions][doc-perms] that the extension needs.
+- The files assigned as the extension service worker, the popup HTML file, the options page, the content scripts, etc.
 
-{% endColumn %}
+The [Manifest keys][doc-manifest] article contains the complete list of default and optional keys. For copy-paste-ready code samples, check out the [Manifest examples][doc-manifest-examples].
 
-{% endColumns %}
+### The extension service worker {: #background_script }
 
-See [Give users options][docs-options] to learn more.
+An extension service worker is an event-based script that the browser runs in the background. It is often used to process data, coordinate tasks in different parts of an extension, and as an extension's event manager. For example, the service worker can listen for and react to events when the extension is first installed, a new tab is created, a new bookmark is added, the extension toolbar icon is clicked, etc.
```

Comment:
Where do we mention that service workers don't need to be registered in extension HTML pages? People seem to assume that they are and we had another issue (#4800) opened on this.

Reply:
@jpmedley I think this is out of scope. We should create a separate PR to address this.

Reply:
I've assigned #4800 to you.